test/athens/common_events/page_test.clj

```diff
@@ -107,3 +107,126 @@
                   (d/q query
                        @@fixture/connection
                        block-uid))))))
+
+
+(test/deftest page-shortcut
+  (test/testing "Add page shortcut"
+    (let [test-uid           "test-page-uid-1"
+          test-title         "test page title 1"
+          create-page-event  (common-events/build-page-create-event -1 test-uid test-title)
+          create-page-txs    (resolver/resolve-event-to-tx @@fixture/connection
+                                                           create-page-event)
+          add-shortcut-event (common-events/build-page-add-shortcut -1 test-uid)
+          add-shortcut-txs   (resolver/resolve-event-to-tx @@fixture/connection add-shortcut-event)]
+
+      ;; create a new page
+      (d/transact @fixture/connection create-page-txs)
+      ;; add the new page to the left sidebar
+      (d/transact @fixture/connection add-shortcut-txs)
+
+      (test/is
+        (not-empty (d/q '[:find (pull ?e [:page/sidebar])
+                          :in $ ?uid
+                          :where
+                          [?e :block/uid ?uid]]
+                        @@fixture/connection test-uid)))))
+
+  (test/testing "Remove page shortcut"
+    (let [test-uid              "test-page-uid-1"
+          test-title            "test page title 1"
+          create-page-event     (common-events/build-page-create-event -1 test-uid test-title)
+          create-page-txs       (resolver/resolve-event-to-tx @@fixture/connection
+                                                              create-page-event)
+          add-shortcut-event    (common-events/build-page-add-shortcut -1 test-uid)
+          add-shortcut-txs      (resolver/resolve-event-to-tx @@fixture/connection add-shortcut-event)
+          remove-shortcut-event (common-events/build-page-remove-shortcut -1 test-uid)
+          remove-shortcut-txs   (resolver/resolve-event-to-tx @@fixture/connection remove-shortcut-event)]
+
+      ;; create a new page
+      (d/transact @fixture/connection create-page-txs)
```

Comment:
Please check after each `d/transact` that the state of DB is what we expect it to be.